examples/custom-resources/service-insight/README.md

```diff
@@ -0,0 +1,189 @@
+# Support for Service Insight
```

Comment:
**Suggestion:** We should probably highlight that this functionality over a https endpoint as its highlighted on https://docs.nginx.com/nginx-ingress-controller/logging-and-monitoring/service-insight/#enabling-service-insight-endpoint

Reply:
added a new example with TLS support